app/controllers/contact_infos_controller.rb

```diff
@@ -41,7 +41,7 @@ def resend_confirmation
                   path = current_user.is_email_pending? ? verification_sent_path :
                                                           profile_path(active_tab: :email)
```

Comment:
Instead of trying to guess the path I think `redirect_to :back` actually does the right thing so I'll do that.